app/Platform/Commands/ResetPlayerAchievement.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Commands;
+
+use App\Platform\Actions\ResetPlayerProgress as ResetPlayerProgressAction;
+use App\Platform\Models\PlayerAchievement;
+use App\Site\Models\User;
+use Exception;
+use Illuminate\Console\Command;
+
+class ResetPlayerAchievement extends Command
+{
+    protected $signature = 'ra:platform:player:reset-achievement
+                            {userId : User ID or username. Usernames containing only numbers are ambiguous and must be referenced by user ID}
+                            {achievementIds : Comma-separated list of achievement IDs}';
+    protected $description = 'Reset achievement(s) for user';
+
+    public function __construct(
+        private readonly ResetPlayerProgressAction $resetPlayerProgress,
+    ) {
+        parent::__construct();
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function handle(): void
+    {
+        if (!app()->environment('local')) {
+            $this->error('This command is only allowed in the local environment');
```

Comment:
Why is it restricted to local?

Reply:
It was a consideration, given the command is completely destructive and leaves no audit trail.